drivers/sensor/amg88xx/amg88xx.h

```diff
@@ -68,6 +66,13 @@
 #define AMG88XX_WAIT_MODE_CHANGE_US	50000
 #define AMG88XX_WAIT_INITIAL_RESET_US	2000
 
+struct amg88xx_config {
+	char *i2c_name;
+	char *gpio_name;
+	u8_t gpio_pin;
```

Comment:
Wrap gpio_name and gpio_pin with `#ifdef CONFIG_AMG88XX_TRIGGER`

Reply:
fixed